site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -201,6 +201,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.tolerations`                           | Allocator [toleration][toleration] labels for pod assignment                                                                                                                                                            | `[]`                               |
 | `agones.allocator.affinity`                              | Allocator [affinity][affinity] settings for pod assignment                                                                                                                                                              | `{}`                               |
 | `agones.allocator.annotations`                           | [Annotations][annotations] added to the Agones allocator pods                                                                                                                                                           | `{}`                               |
+| `agones.allocator.labels`                           | [Labels][labels] added to the Agones allocator pods                                                                                                                                                           | `{}`                               |
```

Comment:
This table will need to added to the feature shortcoded table at the bottom so they don't show up in documentation when merged into `main` (docs get updated immeadiately)
https://agones.dev/site/docs/contribute/#within-a-page

https://github.com/googleforgames/agones/pull/2961 is a good example

Reply:
I've updated the helm file, please review and let me know if there's anything that needs further attention..